cinch/roles/nginx/tasks/selinux.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: enable selinux overrides
+  seboolean:
+    name: "{{ item }}"
+    state: yes
```

Comment:
Use of yes/no in place of true/false is a feature of YAML 1.1 that is not available in 1.2. It doesn't have anything in particular to do with Ansible. Throughout the rest of the playbooks we use true/false in order to avoid these YAML 1.1/1.2 inconsistencies.